### PR TITLE
Update jhub apps related docs

### PR DESCRIPTION
**Pull request by kcpevey**
This PR provides updates to all the jhub-apps related UI elements. 

I've also removed some of the app creation workflow and instead pointed to the jhub-apps docs in order to reduce maintenance here. 

## Reference Issues or PRs

Resolves https://github.com/nebari-dev/nebari-docs/issues/441

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [x] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [x] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [x] This content adheres to the Nebari style guides.

#### Non-text content

- [x] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

**Comment by viniciusdc** [comment]
@kcpevey is this ready for review?

**Comment by kcpevey** [comment]
@viniciusdc yes, its ready for review

File: docs/docs/tutorials/create-dashboard.md [hunk 1]
```diff
@@ -129,30 +125,24 @@ You can run all the cells in your notebook and view the Panel dashboard using th
...
-2. In the app creation interface, enter or select the following:
-   - **Display Name** - Provide meaningful name for your application
-   - **Description (optional)** - Add addition information about the application
-   - **Thumbnail (optional)** - Choose a meaningful thumbnail for your application. The default thumbnail is the application framework's logo.
-   - **Framework** - Select the framework used by your application. For this tutorial, select Panel.
-   - **Filepath** - Path (from root in JupyterLab) to your application code file. For this tutorial, path to the Jupyter Notebook.
-   - **Conda Environment** - Same [environment](#1-create-environment-and-notebook) used while developing your notebook/script which has `jhub-apps` and the corresponding framework.
-   - **Spawner profile** - Instance type (i.e. machines with CPU/RAM/GPU resources) required for running your application.
-   - **Allow Public Access** - Toggle to share the application with your team.
+<Tabs>
```
Comment (pavithraes) [inline docs/docs/tutorials/create-dashboard.md:130]: This looks much cleaner, thank you!

**Review by pavithraes — APPROVED** [review]
This PR looks great! Thanks, Kim. :)

I only have one non-blocking comment, so I'll approve and you can self-merge as you see fit. :)

File: docs/docs/how-tos/jhub-app-launcher.md [hunk 2]
```diff
@@ -33,8 +39,13 @@ JHub App Launcher is enabled by default in the Nebari after version
 [2023.12.1](https://github.com/nebari-dev/nebari/releases/tag/2023.12.1).
```
[the remark this answers is not in the document]
Comment (kcpevey) [reply docs/docs/how-tos/jhub-app-launcher.md]: Yeah that's a good idea. Fixed.

**Pull request merged**